facade/src/main/java/com/redhat/lightblue/migrator/facade/proxy/FacadeProxyFactory.java

```diff
@@ -100,12 +101,12 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
 
                 switch (t) {
                     case LEGACY: {
-                        log.debug("Not a facade operation, proxy passing to legacy");
+                        log.debug("Method \"{}\" is explicitly annotated to call legacy", method);
```

Comment:
Is debug okay for this kind of information?  Do we have debug enabled in dev?  It may be hard to track these down / monitor them.